todo_app/tests/end2end_tests/end2end_test.py

```diff
@@ -0,0 +1,71 @@
+import os, time
+from threading import Thread
+from pytest import fixture
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+from todo_app.data.TrelloClient import TrelloClient
+from todo_app.app import create_app
+
+@fixture(scope='module')
+def app_with_temp_board():
+    apiClient = TrelloClient()
+    # Create the new board & update the board id environment variable
+    response = apiClient.createNewBoard('End_To_End_Test')
+    board_id = response.json()['id']
+    os.environ['BOARD_ID'] = board_id
+    apiClient.updateConfig(board_id)
+    # construct the new application
+    application = create_app()
+    # start the app in its own thread.
+    thread = Thread(target=lambda:application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+    # Tear Down
+    thread.join(1)
+    apiClient.removeBoard(board_id)
+
+
+@fixture(scope="module")
+def driver():
+    chrome_options = Options()
+    chrome_options.add_experimental_option("excludeSwitches", ['enable-automation'])
+    chrome_options.binary_location=r'/usr/bin/google-chrome-stable'
```

Comment:
Worth checking if this is necessary for running these tests on your machine; this makes it OS specific, whereas leaving this empty should be able to find chrome in most standard places